Citavi 5 XML.js

```diff
@@ -0,0 +1,980 @@
+		if (periodicalID) {
+			var periodical = doc.getElementById(periodicalID);
+			item.publicationTitle = ZU.xpathText(periodical, './Name');
+			item.ISSN = ZU.xpathText(periodical, './ISSN');
+			item.journalAbbreviation = ZU.xpathText(periodical, './StandardAbbreviation') ||
+				ZU.xpathText(periodical, './UserAbbreviation1') ||
+				ZU.xpathText(periodical, './UserAbbreviation2');
+		}
+		
+		var authors = ZU.xpathText(doc, '//ReferenceAuthors/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		attachPersons(doc, item, authors, "author");
+		var editors = ZU.xpathText(doc, '//ReferenceEditors/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		attachPersons(doc, item, editors, "editor");
+		var collaborators = ZU.xpathText(doc, '//ReferenceCollaborators/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		attachPersons(doc, item, collaborators, "contributor");
+		var organizations = ZU.xpathText(doc, '//ReferenceOrganizations/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		attachPersons(doc, item, organizations, "contributor");
+		
+		var publishers = ZU.xpathText(doc, '//ReferencePublishers/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		if (publishers && publishers.length>0) {
+			item.publisher = attachName(doc, publishers).join('; ');
+		}
+		
+		var keywords = ZU.xpathText(doc, '//ReferenceKeywords/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		if (keywords && keywords.length>0) {
+			item.tags = attachName(doc, keywords);
+		}
+		if (rememberTags[item.itemID]) {
+			for (var j=0; j<rememberTags[item.itemID].length; j++) {
+				item.tags.push(rememberTags[item.itemID][j]);
+			}
+		}
+		
+		//For all corresponding knowledge items attach a note containing
+		//the information of it.
+		var citations = ZU.xpath(doc, '//KnowledgeItem[ReferenceID="'+item.itemID+'"]');
+		for (var j=0; j<citations.length; j++) {
+			var noteObject = {};
+			noteObject.id = ZU.xpathText(citations[j], '@id');
+			var title = ZU.xpathText(citations[j], 'CoreStatement');
+			var text = ZU.xpathText(citations[j], 'Text');
+			var pages = extractPages(ZU.xpathText(citations[j], 'PageRange'));
+			noteObject.note = '';
+			if (title) {
+				noteObject.note += '<h1>' + title + "</h1>\n";
+			}
+			if (text) { 
+				noteObject.note += "<p>" + ZU.xpathText(citations[j], 'Text') + "</p>\n";
+			}
+			if (pages) {
+				noteObject.note += "<i>" + pages + "</i>";
+			}
+			if (rememberTags[noteObject.id]) {
+				noteObject.tags = rememberTags[noteObject.id];
+			}
+			if (noteObject.note != "") {
+				item.notes.push(noteObject);
+			}
+		}
+		
+		//Locations will be saved as URIs in attachments etc.
+		var locations = ZU.xpath(doc, '//Locations/Location[ReferenceID="'+item.itemID+'"]');
+		for (var j=0; j<locations.length; j++) {
+			var address = ZU.xpathText(locations[j], 'Address');
+			var addressType = ZU.xpathText(locations[j], 'MirrorsReferencePropertyId');
+			if (address) {
+				if (addressType == "Doi" && !item.DOI) {
+					item.DOI = address;
```

Comment:
how about adding the DOI to Extra for items that aren't journal articles or conference papers? (same above)